package.json

```diff
@@ -5,6 +5,7 @@
     "access": "public",
     "provenance": true
   },
+  "type": "module",
```

Comment:
While this is technically a breaking change, since we only publish the types this value doesn't really have much effect at all

Reply:
It might have had an effect after all, see #650 

@wolfy1339 @oscard0m